README.md

```diff
@@ -1,2 +1,125 @@
+as the different inputs required: the user requirements, the system test cases provided as example, the test scenarios
+used as input and the scenario used as example. The replication package also provides the different outputs of our
+exploratory study in the `\docs` folder, the original raw data is available in the [ZENODO](TO-DO) repository.
+
+## Replication package structure and naming conventions:
+
+The naming conventions are :
+
+- **Test Scenarios given as output**  are named using the number of research question as well as the OpenAI model and
+  prompting technique (e.g., RQ1-TestScenarios-GPT4oCOT).
+- **System Test Cases given as output**  are named using the number of research question, followed by a traceability
+  letter (
+  e.g. A,B,C or D) and the test case requested (e.g., RQ2-B-AccessCourseViewClasses).
+- **Prompt Inputs** are named with the word `input*` followed by the type of input (e.g. inputSystemTestCases.txt,
+  inputTestScenarioExample.txt etc.)
+
+The replication package is structured as follows:
+
+1. `/docs`: contains the experimental outputs as well as the experimental baselines, namely as aforementioned .
+
+2. `retorch-llm-rp/src/main`: contains all the necessary Java scripting code for execute the different prompts to the
+   OpenAI API.
+
+3. `retorch-llm-rp/src/main/resources`: contains all the necessary inputs for the prompts: scenarios, test cases and
+   user requirements as well as the examples.
+
+## Experimental Subject
+
+The experimental subject is a real-world application
+called [FullTeaching](https://github.com/codeurjc-students/2019-FullTeaching/tree/Angular-Refactor), used as a
+demonstrator of the [ElasTest EU Project](https://elastest.eu/). FullTeaching provides an education platform composed
+of several test resources, such as web servers, databases, and multimedia servers that allows to create online
+classrooms, classes or publish and create class resources.
+
+To the best of our knowledge, FullTeaching has two test suites available in different
+repositories [[1]](https://github.com/elastest/full-teaching) [[2]](https://github.com/codeurjc-students/2019-FullTeaching/tree/Angular-Refactor).
+The test suite used to generate the raw datasets provided in this replication package is a compilation of the available
+test, cases in these repositories. The test suite is made available as the version TO-DO in
+the [retorch-st-fullteaching](https://github.com/giis-uniovi/retorch-st-fullteaching) GitHub repository.
+
+The user requirements are extracted of the FullTeaching documentation (Fuente Pérez, P. (2017). FullTeaching :
+Aplicación Web de docencia con videoconferencia.) and translated to english. The spanish version can be
+consulted [here](/retorch-llm-rp/src/main/resources/input/inputUserRequirements_spa.txt) and the english
+version [here](/retorch-llm-rp/src/main/resources/input/inputUserRequirements_en.txt).
```

Comment:
Although absolute paths may work, better to use relative paths. Check other places where you also use absolute paths